core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -4308,4 +4308,5 @@ groups:
     - periodic-ipi-deprovision
     paths:
     - .*-postsubmits.yaml$
+    - openshift/os/.*-presubmits.yaml$
```

Comment:
Does this need to be `openshift-os-.*presubmits.yaml` ?  I.e. `-` instead of `/`?

Reply:
It is the regex to match the path (relative to `ci-operator/jobs`).